docs/index.md

```diff
@@ -42,53 +40,50 @@ See [Building with Vanilla](/building-vanilla) and [Customising Vanilla](/custom
   </div>
 </div>
 
-<hr class="is-deep">
+<br>
 
-<div class="u-equal-height">
+<div class="row u-equal-height">
```

Comment:
`u-equal-height` isn't needed here and is causing other problems

Reply:
Ok, will remove 👍